geomstats/geometry/skew_symmetric_matrices.py

```diff
@@ -25,11 +30,24 @@ def __init__(self, n):
         self.basis = gs.zeros((dimension, n, n))
         loop_index = 0
 
-        for row in gs.arange(n - 1):
-            for col in gs.arange(row + 1, n):
-                self.basis[loop_index, row, col] = 1
-                self.basis[loop_index, col, row] = -1
-                loop_index += 1
+        if tests.tf_backend():
```

Comment:
- Maybe create a method: `canonical_basis`?

- We should also avoid testing for backend in the code (`tf_backend()`): can't we abstract this by calling only backend functions?

Reply:
in the long term yes, maybe once we have a `get_from_sparse` for all backends, or stg similar with assignements